src/main/java/core/basesyntax/FruitShop.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.handlers.BalanceOperationHandler;
+import core.basesyntax.handlers.PurchaseOperationHandler;
+import core.basesyntax.handlers.ReturnOperationHandler;
+import core.basesyntax.handlers.SupplyOperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.services.DataProcessorService;
+import core.basesyntax.services.FileReaderService;
+import core.basesyntax.services.FileWriterService;
+import core.basesyntax.services.OperationProcessor;
+import core.basesyntax.services.ReportCreatorService;
+import core.basesyntax.services.impl.DataProcessorServiceImpl;
+import core.basesyntax.services.impl.FileReaderServiceImpl;
+import core.basesyntax.services.impl.FileWriterServiceImpl;
+import core.basesyntax.services.impl.OperationProcessorImpl;
+import core.basesyntax.services.impl.ReportCreatorServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShop {
+    private static final String INPUT_FILE_PATH = "src/main/java/resources/input.csv";
+    private static final String OUTPUT_FILE_PATH = "src/main/java/resources/output.csv";
```

Comment:
Ensure that the file paths are relative and placed in the `src/main/resources` folder as recommended in the checklist.